article/Rougier-2017.md

```diff
@@ -0,0 +1,159 @@
+
+Last, we added a threshold parameter that allows to perform a pre-processing of
+the densty image. Any pixel with a gray level above the threshold is set to the
+threshold value before normalizing the density image. This was necessary for
+replicating some of the original images (see results section for further
+explanation).
+
+# Results
+
+We display only the output of our replication to be compared with the ones in
+the original articles. The climbing shoe (figure @fig:shoe), the corn plant
+(figure @fig:corn) and the large Peperomia plants (figure @fig:large-plant) are
+very similar to the images displayed in the original article. It it to be noted
+that we used different thresholds in order to obtain white areas similmar to
+the original images. Without such threshold there would be no reason to have
+white areas as for example in the leaves of figure @fig:large-plant.
+
+For the small Peperomia (figure @fig:small-plant), the output of our
+replication is less similar. If the hard edges are maintained by the stipple
+drawing, the differences in shading in our replication is not obvious compared
+to the original image. Most probably, the limited resolution of the input image
+may be a limiting factor but it is not clear if the author used these small
+resolution versions or if he used higher resolutions. We tested this hypothesis
+using a bigger image showing leaf with different shading (figure @fig:leafs)
+and in such a case, shading is visible in the output.
+
+
+
+![Climbing shoe (1300x1300), 5 000 dots, 50 iterations, fixed point size](./shoe_1300x1300_org-stipple.pdf){#fig:shoe}
+
+![Corn plant (991x934), 20 000 dots, 50 iterations, fixed point size](./plant4h-stipple.pdf){#fig:corn}
+
+![Large Peperomia plant (700x700), 20 000 dots, 50 iterations, fixed point size](./plant5_700x700-stipple.pdf){#fig:large-plant}
+
+![Small Peperomia plant (400x400), 20 000 dots, 50 iterations, fixed point size](./plant2_400x400-stipple.pdf){#fig:small-plant}
+
+![Figure (512x512), 1000 stipples, 50 iterations, fixed point size](./figure-montage.png){#fig:figure}
+
+![Close-up of large Peperomia leaves (1024x1024), 20000 stipples, 50 iterations, fixed point size](./leaves-montage.png){#fig:leaves}
+
+
+We also provide a new set of data that is freely usable for future comparison
+(CC0 licence).
+
+![Boots (1280x853), 20 000 dots, 50 iterations, variable point size](./boots-montage.png){#fig:boots}
+
+![Pot plant (1195x1024), 20 000 dots, 50 iterations, variable point size](./pot-plant-montage.png){#fig:pot-plant}
+
+![Leafs (1195x1024), 20 000 dots, 50 iterations, variable point size](./leafs-montage.png){#fig:leafs}
+
+
+# Conclusion
+
+Most of the results have been replicated even though some slight discrepancies
+remain in the final output for one image. Without further contact with the
+original author, it is difficult to identify the precise cause but most likely,
+the problem occurs because of the limited resolution of the input picture. We
```

Comment:
"Without further contact with the original author ..." This raises questions; I'd either ask him or just say "It is difficult to ...".